scripts/changelog.py

```diff
@@ -56,7 +56,7 @@ def convert_message(message: str) -> Optional[str]:
 
 
 def get_message_from_pr(repo: str, pr_id: str) -> str:
-    service = GithubService(token=os.getenv("GITHUB_TOKEN"))
+    service = GithubService(token=getenv("INPUT_TOKEN") or getenv("GITHUB_TOKEN"))
```

Comment:
I can't find the `input_token` to be defined or documented anywhere :/

Reply:
ah, finally understood: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#example-specifying-inputs

Reply:
and the commit body has it, I'm good